domains/caiden.xyz.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Caiden-P",
+        "email": "iididhejejdj@gmail.com"
+    },
+    "record": {
+        "TXT": ".\"
```

Comment:
TXT records alone do nothing (also invalid JSON, the `\` escapes the closing `"`)